packages/kuma-gui/src/app/policies/views/PolicyListView.vue

```diff
@@ -142,7 +142,7 @@
                         #role="{ row: item }"
                       >
                         <template
-                          v-if="item.role === 'producer'"
+                          v-if="['producer', 'consumer', 'system', 'workload-owner'].includes(item.role)"
```

Comment:
Should we move this to the data layer now? i.e. if its not one of these values, then default to `system`, then throughout the application we know for sure it can only ever be one of these values?

Reply:
ok lets go with this for now 👍